src/components/Popover/index.jsx

```diff
@@ -1,300 +1,57 @@
+        placement={placement}
+        interactive
+        appendTo={appendTo}
+        onClickOutside={onClickOutside || (() => {})}
+        plugins={[hideOnEsc]}
+        ignoreAttributes
+        zIndex={FDS.ZINDEX_MODAL}
+        moveTransition="transform 0.2s ease-out"
+        content={children}
       >
-        <Popper
-          innerRef={(node) => {
-            refContent.current = node;
-          }}
-          modifiers={popperModifiers}
-          placement={getPopperPlacement(position, alignment)}
-        >
-          {({ placement, ref, style }) => (
-            <div
-              ref={ref}
-              style={{
-                ...contentStyle,
-                ...style,
-              }}
-              data-placement={placement}
-            >
-              {children}
-            </div>
-          )}
-        </Popper>
-      </CSSTransition>
-    );
-
-    return (
-      <Manager>
-        <Reference>
-          {({ ref }) => (
-            <div ref={ref} aria-haspopup="true" aria-expanded={isActive.toString()}>
-              <div ref={refTriggerWrap}>{clonedTrigger}</div>
-            </div>
-          )}
-        </Reference>
-        {disablePortal
-          ? popperContent
-          : ReactDOM.createPortal(
-              popperContent,
-              document.body /* eslint-disable-line no-undef */
-            )}
-      </Manager>
-    );
-  }
-);
-
-Popover.displayName = 'Popover';
-
-const validRef = PropTypes.oneOfType([
-  PropTypes.func,
-  PropTypes.shape({ current: PropTypes.any }),
-]);
+        {triggerElement}
+      </Tippy>
+    </>
+  );
+};
 
 Popover.propTypes = {
-  /** JSX - Acts as a positioning reference for the popover and triggers active state */
-  trigger: PropTypes.oneOfType([PropTypes.node, PropTypes.element]).isRequired,
-
-  /**
-   * JSX - Content to place in the positioned popover container. The popover container
-   * does not provide any default styling; content should be styled with background,
-   * borders, and shadows as appropriate.
-   */
-  children: PropTypes.oneOfType([PropTypes.node, PropTypes.element]).isRequired,
-
-  /**
-   * What mode of interaction triggers the active/inactive state of the popover.
-   *
-   * `hover` - popover opens on trigger hover/focus
-   *
-   * `click` - popover opens on trigger click
-   *
-   * `controlled` - enables "fully controlled" mode in which the popover is only active
-   * when the `isOpen` prop is set to `true`. Use `onUserDismiss` to handle dismissal
-   * events from user interaction.
-   */
-  interactionMode: PropTypes.oneOf(VALID_INTERACTION_MODES),
-
-  /** Controlls active state of popover when in fully controlled interaction mode */
+  appendTo: PropTypes.oneOfType([PropTypes.element, PropTypes.string]),
+  children: PropTypes.node,
+  /** Element that popover is positioned on */
+  triggerElement: PropTypes.node,
+  placement: PropTypes.oneOf(['auto', 'top', 'right', 'bottom', 'bottom-start', 'left']),
```

Comment:
it may be worth looking at the current placement for popover and tooltip and seeing if we want to align them for whatever reason. The only reason they can't be aligned now is popover supports `bottom-start` which we know is the default way we want popovers rendered. That being said, I don't want to seek consistency with Tooltip just for DRY sake... if we don't ever want to position tooltip with a bottom-start, I think these should stay separate and that it's totally fine to keep separate.